openreview/venue/invitation.py

```diff
@@ -5052,4 +5052,127 @@ def set_edit_venue_group_invitations(self):
                     }
                 }
             )
-        )        
+        )
+
+    def set_venue_template_invitations(self):
+
+        super_id = 'openreview.net'
```

Comment:
Should this be 'OpenReview.net' for prod?

Reply:
thanks for the catch, let me fix it.